andela/mag-ah-frontend
# #161256141 Application should deploy on Heroku

**What does this PR do?**
Enable the application to deploy on Heroku

**Description of Task to be completed?**
Remove proxy from the package.json
Add BASE_URL environmental variable

**How should this be manually tested?**
Go to [https://magnificent6-frontend.herokuapp.com/ ](`https://magnificent6-frontend.herokuapp.com/ `)
Click on Get started
Fill in all the fields in the form
It should return a success message

**What are the relevant pivotal tracker stories?**
[#161256141](https://www.pivotaltracker.com/story/show/161256141)

**Screenshots**

Signup form
<img width="824" alt="screen shot 2018-10-17 at 10 45 38" src="https://user-images.githubusercontent.com/12625365/47070722-8351b780-d1fa-11e8-9773-a42b2d5805aa.png">

Success Message
<img width="739" alt="screen shot 2018-10-17 at 10 46 00" src="https://user-images.githubusercontent.com/12625365/47070745-906ea680-d1fa-11e8-9e3a-6921c66b903b.png">
